src/Debug/Model/ServiceInfo.php

```diff
@@ -28,21 +30,28 @@ public function getReferences(): array
         $refs = [];
 
         foreach ($this->arguments as $arg) {
-            if ('reference' === $arg->type && !\is_array($arg->value)) {
-                $refs[] = ltrim((string) $arg->value, '@');
-
-                continue;
```

Comment:
Why was this removed?

Reply:
At thid position a continue makes no sense because it it the end of the loop.
At the if condition before a continue could be set but makes IMHO the code not more readable...